server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -387,8 +388,36 @@ public void getQuotaShouldReturnBothWhenValueSpecified() throws Exception {
                 .extract()
                 .jsonPath();
 
-        assertThat(jsonPath.getLong(SIZE)).isEqualTo(maxStorage);
-        assertThat(jsonPath.getLong(COUNT)).isEqualTo(maxMessage);
+        SoftAssertions softly = new SoftAssertions();
+        softly.assertThat(jsonPath.getLong("computed." + SIZE)).isEqualTo(42);
+        softly.assertThat(jsonPath.getLong("computed." + COUNT)).isEqualTo(52);
+        softly.assertThat(jsonPath.getLong("user." + SIZE)).isEqualTo(42);
+        softly.assertThat(jsonPath.getLong("user." + COUNT)).isEqualTo(52);
+        softly.assertThat(jsonPath.getLong("global." + SIZE)).isEqualTo(1111);
+        softly.assertThat(jsonPath.getLong("global." + COUNT)).isEqualTo(22);
+    }
+
+    @Test
+    public void getQuotaShouldReturnOnlySpecifiedValues() throws Exception {
+        maxQuotaManager.setDefaultMaxStorage(QuotaSize.size(1111));
+        maxQuotaManager.setMaxMessage(userQuotaRootResolver.forUser(BOB), QuotaCount.count(52));
+
+        JsonPath jsonPath =
+            given()
+                .get(QUOTA_USERS + "/" + BOB.asString())
+                .then()
```

Comment:
Ident? By convention, `then()` should be on the same line than `given()`...

Reply:
ok